packages/web-components/fast-components/src/carousel/test/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./carousel-test-slide";
```

Comment:
Can you change the name of the test folder to `__test__` so that it matches our other test folder name, and change the `.npmignore` to:
`dist/**/__test__/` instead of `dist/dts/__test__/` and `dist/esm/__test__/`